### ISSUE TITLE
Primary Key missing in auto-generated migration

### ISSUE THREAD

**Issue by adriangb** [piccolo-orm/piccolo#170]
Using Postgres & Piccolo 0.32.0, the auto-generated migrations fail to include the primary key.

Schema:

```python
from piccolo.table import Table
from piccolo.columns import ForeignKey, UUID

class Types(Table):
    id = UUID(primary_key=True)

class Things(Table):
    type = ForeignKey(references=Types)
```

When I try to run migrations via:

```
piccolo migrations new my_app --auto
piccolo migrations forwards all
```

I get:

```
Running MigrationManager ...
The command failed.
foreign key constraint "things_type_fkey" cannot be implemented
DETAIL:  Key columns "type" and "id" are of incompatible types: integer and uuid.
```

Looking at the generated migration, I see:

```python
class Types(Table, tablename="types"):
    pass
```

Adding the PK manually to the migration fixes the issue:

```python
class Types(Table, tablename="types"):
    id = UUID(primary_key=True)
```

**Comment by aminalaee** [comment]
@adriangb Thank you for reporting this.
There's an open issue to fix this #156 . Unfortunately the fix is not backwards compatible (yet) but we're trying to fix that.

**Comment by adriangb** [comment]
Nice! Feel free to close this if it's a duplicate

**Comment by aminalaee** [comment]
@adriangb That's fine I think. We didn't really have an issue for that PR. Thanks for reporting it.

**Comment by dantownsend** [comment]
I think the issue here is the `ForeignKey` should be created with type `UUID`, otherwise it can't store the reference to a `UUID` primary key.

This should work - I'll investigate what went wrong.

**Comment by dantownsend** [comment]
I've been able to replicate the bug locally, and know what the issue is.

**Comment by dantownsend** [comment]
@adriangb Thanks for reporting this, and @aminalaee for the assistance. Should be fixed on PyPI now (v0.33.0) - let me know if there are still issues.
